plutus-core/plutus-core/src/PlutusCore/MkPlc.hs

```diff
@@ -69,13 +72,25 @@ class TermLike term tyname name uni fun | term -> tyname name uni fun where
     termLet = mkImmediateLamAbs
     typeLet = mkImmediateTyAbs
 
--- | Embed a type from a universe into a PLC type.
+-- TODO: make it @forall {k}@ once we have that.
```

Comment:
What do these two TODOs 'mean'?

Reply:
[This](https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0099-explicit-specificity.rst).

I'll expand the comment.